packages/salesforcedx-vscode-apex-replay-debugger/test/vscode-integration/commands/quickLaunch.test.ts

```diff
@@ -12,6 +12,7 @@ import { MockTestOrgData, testSetup } from '@salesforce/core/lib/testSetup';
 import { notificationService } from '@salesforce/salesforcedx-utils-vscode/out/src/commands';
 import { TraceFlags } from '@salesforce/salesforcedx-utils-vscode/out/src/helpers';
 import * as utils from '@salesforce/salesforcedx-utils-vscode/out/src/index';
+import { SFDX_CORE_CONFIGURATION_NAME } from '@salesforce/salesforcedx-utils-vscode/out/src/index';
```

Comment:
since `@salesforce/salesforcedx-utils-vscode/out/src/index` is already being imported, can line 15 be removed and on line 54, use `utils.SFDX_CORE_CONFIGURATION_NAME`?